### PR TITLE
2.x Bug fix Back-ports

### PR DESCRIPTION
**Pull request by Eric-Vin**

### CODE REVIEW

**Comment by Eric-Vin** [comment]
This passes tox on my machine, so we should be good.

File: tests/simulators/carla/test_carla.py [hunk 1]
```diff
@@ -10,6 +10,27 @@ def test_basic(loadLocalScenario):
...
+    code = f"""
+        param map = r'{mapPath}'
+        param carla_map = 'Town01'
+        model scenic.simulators.carla.model
+        action = SetGearAction(0)
+        ego = Car
+        assert action.canBeTakenBy(ego)
+    """
+    for _ in range(2):
+        compileScenic(code, mode2D=True)
```
Comment (dfremont) [inline tests/simulators/carla/test_carla.py:31]: Hmm, how did this work under 2.x? There shouldn't be any `mode2D` option. Did you try testing with CARLA installed?

**Review by dfremont — CHANGES_REQUESTED** [review]
Looks good except for one small issue above -- but make sure to test with CARLA installed too.

File: tests/simulators/carla/test_carla.py (hunk 1 above)
Comment (Eric-Vin) [reply tests/simulators/carla/test_carla.py]: Good catch, fixed.

**Comment by Eric-Vin** [comment]
Ah, good point on the Carla testing. I tested with that installed and fixed the relevant test, and we're now passing everything.

**Review by dfremont — APPROVED** [review]

**Pull request merged**